src/main/java/uk/gov/hmcts/darts/annotation/controller/AnnotationController.java

```diff
@@ -43,4 +46,30 @@ public ResponseEntity<Void> deleteAnnotation(Integer annotationId) {
         annotationService.delete(annotationId);
         return ResponseEntity.noContent().build();
     }
+
+    @Authorisation(
+            bodyAuthorisation = true,
```

Comment:
annotationId is in the path, not the body, so `bodyAuthorisation = true,` is not required